src/Microsoft.DotNet.Build.Tasks.Feed.net45/PushToBlobFeed.Desktop.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.Build.Common.Desktop;
+
+namespace Microsoft.DotNet.Build.Tasks.Feed
+{
+    public partial class PushToBlobFeed
+    {
+        static PushToBlobFeed()
```

Comment:
Is this the only task in this library? If so this is good, if not then we should do it for each task.

Reply:
It isn't, but I believe it's the only one that uses Sleet. I agree we should apply it to all though, for maintainability/safety--I already filed https://github.com/dotnet/buildtools/issues/1944 for that.

Reply:
Ah, it looks like `GetBlobFeedPackageList` uses Sleet too--I'll work on getting 1944 through. This fix should be sufficient for the Core repos, though.

FYI @mmitche, this version might still not work for DotNet-Orchestration-Utilities since it uses more of these Feed tasks than normal repos do.

Reply:
Okay.  We are still on an older version in that repo so it may be okay for now?

Reply:
Yep. This problem only started with yesterday's BuildTools builds, and the last BT update the orchestration repo had was a few weeks ago.